js-envs-test-suite/src/test/scala/org/scalajs/jsenv/test/RetryingComJSEnvTest.scala

```diff
@@ -27,17 +26,17 @@ class RetryingComJSEnvTest extends JSEnvTest with ComTests {
     private[this] var fails = 0
     private[this] var failedReceive = false
 
-    def jsRunner(libs: Seq[ResolvedJSDependency],
+    def jsRunner(libs: Seq[VirtualJSFile],
```

Comment:
Not directly related to this PR, but now that `LinkingUnitJSEnv` is gone, we should probably rethink whether `loadLibs` still makes sense.